packages/strapi-database/lib/constants/index.js

```diff
@@ -24,6 +24,8 @@ const RESERVED_ATTRIBUTE_NAMES = [
   'attributes',
   'relations',
   'changed',
+  'page',
```

Comment:
This is not necessary as the filters are sent in _where from the content manager & in the generated API there is no pagination